config/sync/jsonapi_extras.jsonapi_resource_config.block--block.yml

```diff
@@ -0,0 +1,51 @@
+uuid: 2763f554-cd5d-4c4e-a9df-4cd228f1a8b2
+langcode: en
+status: true
+dependencies: {  }
+id: block--block
+disabled: false
+path: blocks
+resourceType: blocks
+resourceFields:
+  id:
+    fieldName: id
+    publicName: internalId
+    enhancer:
+      id: ''
+    disabled: false
+  status:
+    fieldName: status
+    publicName: isPublished
```

Comment:
I think its positive to sever the ties between the names in the back-end and the names in the front-end.  As a front-end developer I don't want to know or care about your back-end implementation.

More than that, we want to show people that they don't need to marry a naming pattern from Drupal.